Makefile

```diff
@@ -3,7 +3,7 @@ ORG             := cnabio
 BINDIR          := $(CURDIR)/bin
 GOFLAGS         :=
 GOBUILDTAGS     := osusergo
-LDFLAGS         := -w -s -X github.com/cnabio/signy/pkg/docker.Tag=$(TAG)
+LDFLAGS         := -w -s -X github.com/cnabio/signy/pkg/docker.Tag=latest
```

Comment:
I think a more useful piece of information would be getting the latest commit information - for example, this is how it is done in Duffle:

https://github.com/cnabio/duffle/blob/c796af33bc97092623847599a1ff903bb40244a0/Makefile#L11

And the approach in Porter:

https://github.com/deislabs/porter/blob/93a62ce730ea59be9d1a976626f68c8011faf6e2/Makefile#L7-L8

Reply:
Do we really need it for Signy right now, though? I had previously built the Docker image using the commit hash, and your thinking back then IIRC was that it was building too many unnecessary images.

Reply:
I will merge this PR in the interest of progress. We can revisit this issue should it arise again later.